presto-main/etc/catalog/prometheus.properties

```diff
@@ -1,5 +1,8 @@
 connector.name=prometheus
-prometheus.uri=http://localhost:9090
-prometheus.query-chunk-duration=10m
-prometheus.max-query-duration=1h
+prometheus.uri=https://52.118.211.217:9090
```

Comment:
What is this ip here?

Reply:
Accidentally committed test ip. I removed the line